crates/api_crud/src/user/create.rs

```diff
@@ -111,28 +118,12 @@ impl PerformCrud for Register {
     let local_user_form = LocalUserInsertForm::builder()
       .person_id(inserted_person.id)
       .email(data.email.as_deref().map(str::to_lowercase))
-      .password_encrypted(data.password.to_string())
+      .password_encrypted(hash(&data.password, DEFAULT_COST)?)
```

Comment:
Seems fine to move this up, but I hope you tested.

Reply:
yep

Reply:
This wasnt working after all, because it also needs to hash the setup password defined in config file. Reverted the change.